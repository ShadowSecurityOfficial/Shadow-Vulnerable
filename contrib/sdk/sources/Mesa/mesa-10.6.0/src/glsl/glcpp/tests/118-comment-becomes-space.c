version https://git-lfs.github.com/spec/v1
oid sha256:049983a00572110ad135cb86120907d97bc3707f110b09a4dec52d49542dcc42
size 34
