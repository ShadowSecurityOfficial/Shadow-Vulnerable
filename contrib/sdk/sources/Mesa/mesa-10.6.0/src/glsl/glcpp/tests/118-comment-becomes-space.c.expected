version https://git-lfs.github.com/spec/v1
oid sha256:bd1e8ad0ca122b4042ac4a31376509c54861c453211d775a041b3512f3132f32
size 16
